src/compiler/crystal/tools/playground/server.cr

```diff
@@ -21,11 +21,16 @@ module Crystal::Playground
       instrumented = Playground::AgentInstrumentorTransformer.transform(ast).to_s
       Log.info { "Code instrumentation (session=#{session_key}, tag=#{tag}).\n#{instrumented}" }
 
+      
+      
+      ip = @@host
+      ip = "localhost" if ip == "127.0.0.1" || ip == "localhost" || ip == nil
```

Comment:
I don't think this is correct. You can't assume nil or `127.0.0.1` mean `localhost`.
Also, any kind of modification should happen already at the server, not in each session.

Reply:
> I don't think this is correct. You can't assume nil or `127.0.0.1` mean `localhost`. Also, any kind of modification should happen already at the server, not in each session.
yes, I also don't think is correct , I assume if not specify the ```-b``` then use default ```localhost```. I just try to improve that becuase the raw code is alaways ```localhost```.

Reply:
> I don't think this is correct. You can't assume nil or `127.0.0.1` mean `localhost`. Also, any kind of modification should happen already at the server, not in each session.

sorry，I jgot your idea just now, should modification the ip outside of each session!

Reply:
> I don't think this is correct. You can't assume nil or `127.0.0.1` mean `localhost`. Also, any kind of modification should happen already at the server, not in each session.

I had try to fix it in my second commit~